src/server/scripts/EasternKingdoms/ZulGurub/boss_mandokir.cpp

```diff
@@ -339,6 +474,85 @@ class npc_ohgan : public CreatureScript
     }
 };
 
+class npc_chained_spirit : public CreatureScript
+{
+public:
+    npc_chained_spirit() : CreatureScript("npc_chained_spirit") { }
```

Comment:
Change this to the newer system. Define only the ScriptedAI and then, in AddSC_boss_mandokir, use RegisterCreatureScript(npc_chained_spirit);

Reply:
pending